Makefile

```diff
@@ -374,26 +389,21 @@ clean: ## Clean
 	rm -rf "$(CURDIR)/packages"
 	rm -rf "$(CURDIR)/.open-space-toolkit"
 
-################################################################################################################################################################
-
 .PHONY: pull pull-development-image \
 		build build-images build-development-image \
 		pull-release-images pull-release-image-cpp pull-release-image-python pull-release-image-jupyter \
 		build-release-image-cpp build-release-image-python build-release-image-jupyter \
 		build-documentation build-packages-cpp \
 		start-development start-python start-jupyter-notebook debug-jupyter-notebook \
+		format format-check \
```

Comment:
Need to add format-python?

Reply:
What about updating our Makefiles to add the `.PHONY` after each target (as we do in most projects), to avoid maintaining the laundry list at the end?

Reply:
I like it. Done!